plugins/field-bitmap/src/field-bitmap.ts

```diff
@@ -6,12 +6,20 @@
 
 import * as Blockly from 'blockly/core';
 
+Blockly.Msg['BUTTON_LABEL_RANDOMIZE'] = 'Randomize';
+Blockly.Msg['BUTTON_LABEL_CLEAR'] = 'Clear';
```

Comment:
I hadn't noticed this previously.  I think it's fine as it is for the moment, but the team should probably make some decisions about how we will deal with localisation of plugins now that more formerly-core functionality has been moved out of Blockly.

Reply:
For this, I was following guidance from @maribethb who pointed out that this is how it’s done elsewhere:

https://google.github.io/blockly-samples/plugins/block-plus-minus/README

https://github.com/google/blockly-samples/blob/337ce75ab72a74b6c9c2551f92442cbf848a5e90/plugins/block-plus-minus/src/procedures.js#L15

Naturally, I’m open to changing this if there’s a better way. I had originally set up to take in the translation strings as configuration options (which is why there was ‘buttonOptions’ initially).

Reply:
Her guidance is certainly good.  I've no issue with the code; my comment was more as a reminder to myself that we should probably think about whether we want to add messages like this to the set that get translated by translatewiki. Until now we've only had messages in core Blockly and Blockly Games translated, not messages for plugins—but with some formerly-core fields and blocks now being moved to plugins we should probably revisit that.